<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="vehicle_name"		default="HUACHEN"/> <!--BYD_TANG, HUACHEN-->
  <arg name="listen_port"		  default="8500"/>

  <!--get vehicel ECU data-->
  <!--node pkg="frontal_delphi_radar" type="get_ECU" name="get_ecudata" output="screen">
		<param name="vehicle_name"		value="$(arg vehicle_name)"/>
		<param name="listen_port"		value="$(arg listen_port)"/>
  </node!-->

  <node pkg="frontal_mmw_radar" type="get_delphi_radar_data" name="get_radar_data" output="screen">
  </node>

  <node pkg="frontal_mmw_radar" type="frontal_delphi_radar" name="frontal_delphi_radar" output="screen">
  </node>
  

</launch>
